README.md

```diff
@@ -198,14 +198,139 @@ Example:
 }
 ```
 
+### Filtering Content Items
+
+**Note:** Filtering content via `filterBy() | filterByContentType() | filterByParentId() | filterContentTypes()` is only supported when using [Content Delivery 2](#content-delivery-versions)
+
+Once you have [a schema set up for filtering](https://amplience.com/docs/development/contentdelivery/filterandsort.html), the content item must be published before it can be retrieved using this SDK.
+
+### Constructing a request
+
+The `filterBy() | filterByContentType() | filterByParentId() method will return a instance of the`FilterBy` class which has helper functions to construct a filterBy request.
```

Comment:
the`FilterBy`  - needs a space

Not sure if it's important that they know about the FilterBy class. Seems more important to get across `filterByContentType()` and  `filterByParentId()` are specialist types of `filterBy()`.

Think it should start with giving examples of `filterBy()` then underneath explain 'As Filtering by content type and filtering by a parent id is are common uses of filter by, we have the following helper functions:' etc.

Reply:
yeah that makes sense, on the method docs for filterByContentType() and filterByParentId() i say it's equivalent to the filterBy